app/views/rules_of_origin/steps/_product_specific_rules.html.erb

```diff
@@ -15,4 +15,4 @@
 <% end %>
 
 <%= render 'about_commodity', commodity_code: current_step.commodity_code,
-                              description: current_step.commodity_description %>
+                              description: current_step.declareable_description %>
```

Comment:
I'm not actually sure of the correct spelling but we've used `declarable` instead of `declareable` elsewhere in the codebase so its best to be consistent